src/oop/app/src/Parsers/KinoukrDomCrawlerParserAdapter.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace src\oop\app\src\Parsers;
+
+use src\oop\app\src\Models\Movie;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+class KinoukrDomCrawlerParserAdapter implements ParserInterface
+{
+
+    private string $title = '';
```

Comment:
Do you really need these properties? You can use the local variables instead as you have only one method. But it's not a mistake, only a recommendation.

Reply:
Yes, your right, thank You.
Remove class properties and rewrite method in last commit.